### ISSUE TITLE
why cannot turn off  sonoff from automation?

### ISSUE THREAD

**Issue by violgore** [peterbuga/HASS-sonoff-ewelink#113]
why cannot turn off  sonoff from automation?

I cant in the last update,
 I use 
switch.turn_off   , I try homeassistant.turn_off
but nothing 

Thanks

**Comment by neonwrx** [comment]
such a problem, what can help?

**Comment by violgore** [comment]
'1558179512434':
  alias: PORTERILLO
  sequence:
  - data:
      entity_id: switch.sonoff_1000646fec
    service: switch.turn_on
  - delay: 00:00:01
  - data:
      entity_id: switch.sonoff_1000646fec
    service: switch.turn_off


This is a script, I calle from automation

- id: '1557254386808'
  alias: PORTERILLO ON
  trigger:
  - entity_id: switch.sonoff_1000646fec
    platform: state
    to: 'on'
  condition: []
  action:
  - data: {}
    service: script.1558179512434

When I switch on sonoff , not go to OFF

sorry for my english
Thanks in advanced

**Comment by hdjweb** [comment]
I have the same problem with outgoing home automation to turn off all lights

**Comment by peterbuga** [comment]
pretty much duplicate of https://github.com/peterbuga/HASS-sonoff-ewelink/issues/92

TLDR; ewelink doesn't support toggle of multiple devices in the same time (or very fast), add a 1-2sec (or increase the value) delay between different switches

@violgore in your case you could use the inching option available in the latest firmwares
